sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -1322,24 +1321,15 @@ static void _auto_configure_addr(gnrc_netif_t *netif, const ipv6_addr_t *pfx,
 #endif  /* GNRC_IPV6_NIB_CONF_6LN */
     }
 
-#if GNRC_IPV6_NIB_CONF_6LN
-    if (gnrc_netif_is_6ln(netif)) {
-        /* don't do this beforehand or risk a deadlock:
-         *  * gnrc_netif_ipv6_addr_add_internal() adds VALID (i.e. manually configured
-         *    addresses to the prefix list locking the NIB's mutex which is already
-         *    locked here) */
-        netif->ipv6.addrs_flags[idx] &= ~GNRC_NETIF_IPV6_ADDRS_FLAGS_STATE_MASK;
```

Comment:
FYI: the address flags are initialized TENTATIVE, due to [this line](https://github.com/RIOT-OS/RIOT/pull/8563/files#diff-2b29f6c8fc858131a3d21e4375f5681fR1302).